CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# 0.1.0
+- [feature] account support
```

Comment:
@KgaevskiT can you have another look here, we are very close to pushing that to production.